### DISCUSSION TITLE
Deviation at sample rate

### DISCUSSION THREAD

**Discussion by cmarkus413** [aewallin/allantools#138] [General]
Hello,

I am confused why the Allan deviation at the sample rate is not equal to the standard deviation of the data set. For example:

```
import numpy as np
import allantools

fake_data= np.random.normal(0, 1, 1000)
taus = np.arange(1, int(len(fake_data)/2))
taus, ad, ade, ns = allantools.oadev(fake_data, 1.0, taus=taus)

print(f'stdev: {np.std(fake_data)}')
print(f'adev(1): {ad[0]}')

> stdev: 1.024525966134758
> adev(1): 1.7235083188043996

```

Why is this the case? Calculating manually they should be equal.

**Comment by aewallin** [comment]
The default data_type is phase in units of seconds, so your fake_data is interpreted as phase (in seconds).
It takes three phase-measurements to compute two fractional frequency values, so this leads to a factor sqrt(3) in the ADEV.

If you want to input fractional frequency, use data_type='freq'
or alternatively keep the call to oadev() the same, but convert fake_data to phase with e.g.
fake_data_phase = np.cumsum(fake_data)

**Comment by cmarkus413** [comment]
Thank you for the response! This is very helpful.

Also sorry for putting this in "issues", I'll make sure to post basic questions in discussion next time.
